routes.js

```diff
@@ -77,6 +77,9 @@ module.exports = (app) => {
   // TODO(tony): change it back to /restaurants/me
   router.put("/restaurants/:id", controller.updateRestaurant);
   router.post("/restaurants/register", controller.createRestaurant);
+  router.post("/restaurants/create-group", controller.createRestaurantGroup);
+  router.post("/groups/:groupID/add-to-group", controller.addUserToGroup);
+  router.post("/restaurants/add-to-group/:id", controller.addRestaurantToGroup);
```

Comment:
Maybe to not confuse between restaurant and user, let's do 
`groups/:groupID/add-user`
and
`groups/:groupID/add-restaurant`